base/applications/notepad/lang/ro-RO.rc

```diff
@@ -33,46 +33,46 @@ MAIN_MENU MENU
 BEGIN
     POPUP "&Fișier"
     BEGIN
-        MENUITEM "&Nou\tCtrl+N", CMD_NEW
+        MENUITEM "No&u\tCtrl+N", CMD_NEW
         MENUITEM "Fereastră &nouă\tCtrl+Shift+N", CMD_NEW_WINDOW
-        MENUITEM "&Deschidere…\tCtrl+O", CMD_OPEN
-        MENUITEM "S&alvează\tCtrl+S", CMD_SAVE
-        MENUITEM "Sal&vare în…", CMD_SAVE_AS
+        MENUITEM "&Deschidere...\tCtrl+O", CMD_OPEN
+        MENUITEM "&Salvare\tCtrl+S", CMD_SAVE
+        MENUITEM "S&alvare ca...", CMD_SAVE_AS
         MENUITEM SEPARATOR
-        MENUITEM "Inițiali&zare pagină…", CMD_PAGE_SETUP
-        MENUITEM "I&mprimare…\tCtrl+P", CMD_PRINT
+        MENUITEM "Iniţiali&zare pagină...", CMD_PAGE_SETUP
+        MENUITEM "I&mprimare...\tCtrl+P", CMD_PRINT
         MENUITEM SEPARATOR
-        MENUITEM "I&eșire", CMD_EXIT
+        MENUITEM "&Ieşire", CMD_EXIT
     END
     POPUP "&Editare"
     BEGIN
-        MENUITEM "Anula&re\tCtrl+Z", CMD_UNDO
+        MENUITEM "&Anulare\tCtrl+Z", CMD_UNDO
         MENUITEM SEPARATOR
-        MENUITEM "&Decupează\tCtrl+X", CMD_CUT
-        MENUITEM "&Copiază\tCtrl+C", CMD_COPY
-        MENUITEM "&Lipește\tCtrl+V", CMD_PASTE
-        MENUITEM "Șt&erge\tDel", CMD_DELETE
+        MENUITEM "&Decupare\tCtrl+X", CMD_CUT
+        MENUITEM "&Copiere\tCtrl+C", CMD_COPY
+        MENUITEM "Li&pire\tCtrl+V", CMD_PASTE
+        MENUITEM "Şt&ergere\tDel", CMD_DELETE
         MENUITEM SEPARATOR
-        MENUITEM "&Găsire…\tCtrl+F", CMD_SEARCH
+        MENUITEM "Cău&tare...\tCtrl+F", CMD_SEARCH
         MENUITEM "&Următorul găsit\tF3", CMD_SEARCH_NEXT
-        MENUITEM "Î&nlocuire…\tCtrl+H", CMD_REPLACE
-        MENUITEM "S&alt la…\tCtrl+G", CMD_GOTO
+        MENUITEM "Î&nlocui&re...\tCtrl+H", CMD_REPLACE
+        MENUITEM "Salt &la...\tCtrl+G", CMD_GOTO
         MENUITEM SEPARATOR
-        MENUITEM "Sele&ctare totală\tCtrl+A", CMD_SELECT_ALL
-        MENUITEM "&Dată/Oră\tF5", CMD_TIME_DATE
+        MENUITEM "&Selectare totală\tCtrl+A", CMD_SELECT_ALL
+        MENUITEM "Dată/&Oră\tF5", CMD_TIME_DATE
     END
-    POPUP "F&ormatare"
+    POPUP "F&ormat"
     BEGIN
-        MENUITEM "În&cadrează liniile lungi", CMD_WRAP
-        MENUITEM "&Font…", CMD_FONT
+        MENUITEM "&Încadrare cuvânt", CMD_WRAP
+        MENUITEM "&Font...", CMD_FONT
     END
-    POPUP "&Afișare"
+    POPUP "&Vizualizare"
     BEGIN
         MENUITEM "&Bară de stare", CMD_STATUSBAR
     END
-    POPUP "Aj&utor"
+    POPUP "&Ajutor"
     BEGIN
-        MENUITEM "&Vizualizare Ajutor…", CMD_HELP_CONTENTS
+        MENUITEM "&Termeni din Ajutor", CMD_HELP_CONTENTS
```

Comment:
This lines change is okay. My (german) WinXPSP3 notepad also doesn't have any dots here in the end.